uiowa/uiowa
# Adopt DrupalVM.

# Description
Resolves #976  
Resolves #942 
Resolves #933 
Resolves #919 

# Testing
1. Install VirtualBox 6.0.16
  - https://www.virtualbox.org/wiki/Download_Old_Builds_6_0
2. Install Vagrant 2.2.6
  - https://www.vagrantup.com/downloads.html
3. Comment out any `multisites` local overrides in `blt/local.blt.yml` file.
4. Remove all local setting files with `rm -f docroot/sites/*/settings/local.settings.php docroot/sites/*/local.drush.yml`.
5. Regenerate local settings with `blt blt:init:settings` or `blt bis` or short.
6. Initialize the VM with `blt vm`. Follow any on-screen instructions. There may be warnings/errors but you should get a green message at the end mentioning the dashboard: http://dashboard.local.drupal.uiowa.edu. Visit that URL.
7. Uncomment any `multisites` overrides in `blt/local.blt.yml` file.
8. SSH into the VM with `vagrant ssh` on a new terminal tab.
9. Delete the node_modules directory in the application root.
10. Sync local sites with `blt drupal:sync:all-sites` or `blt dsa` for short. 

Creating a site will require running `vagrant provision` from the host, i.e. your machine. This will create a new VHOST for the domain and it will be listed in the dashboard.


Be sure to scrub any sensitive info, users, pws, server paths, etc. from comments posted here.
